tools/regenerate_dependency_summaries.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+python3 ./tools/dependency_summary.py > ./DEPENDENCIES.md
+python3 ./tools/dependency_summary.py --all-android-targets --package lockbox > megazords/lockbox/DEPENDENCIES.md
+python3 ./tools/dependency_summary.py --all-android-targets --package fenix > megazords/fenix/DEPENDENCIES.md
+python3 ./tools/dependency_summary.py --all-android-targets --package reference-browser > megazords/reference-browser/DEPENDENCIES.md
```

Comment:
Hmm, now that #1103 renames `reference-browser-megazord` to `full-megazord` I guess we'll probably need to track down whoever is including those us transitively to make sure they have this too.

(In that sense, maybe it would have been better for that PR to break them so that they reach out to us :p)